node_agent.spec

```diff
@@ -0,0 +1,65 @@
+Name: tendrl-node-agent
+Version: 0.0.1
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Module for Node Agent
+Source0: %{name}-%{version}.tar.gz
+License: LGPLv2+
+URL: https://github.com/Tendrl/node_agent
+
+BuildRequires: systemd
+BuildRequires: python2-devel
+BuildRequires: python-sphinx
+BuildRequires: pytest
+
+Requires: python-etcd
+Requires: python-gevent
+Requires: python-greenlet
+Requires: python-taskflow
+Requires: collectd
+Requires: python-jinja2
+Requires: tendrl-bridge-common
+Requires: systemd
+
+%description
+Python module for Tendrl node bridge to manage storage node in the sds cluster
+
+%prep
+%setup %{name}-%{version}
```

Comment:
Please just use `%setup` here.

Did you try building this package in mock for EPEL 7?